New artefact test modification (#350)
biometricITC/cPP-biometrics: Supporting Documents/BS_SD.adoc

This Section describes EAs for AVA_VAN.1 step by step following the order of AVA_VAN.1 CEM work units.

===== Search for new artefacts
Each of the supported biometric modalities have a specific set of defined artefacts species in the <<Toolbox>> to be used in testing. These are devised based on publicly available information published by the publication date of the <<Toolbox>>. The BIO-iTC also verifies that test items cover all existing artefact species that are within the scope of Basic attack potential defined in <<Attack Potential and TOE resistance>>.

> However, new artefacts species may be found after the <<Toolbox>> is published. So the evaluator shall search publicly available information that is published after the publication date of the <<Toolbox>> to look for new artefact species. New artefact species are those artefacts that are out of scope of the <<Toolbox>> and need to be made in a completely different way with significantly different materials that are not covered by the <<Toolbox>>.

Reviewer: ```suggestion
However, new artefacts species may be found after the <<Toolbox>> is published. The evaluator shall search publicly available information that is published after the publication date of the <<Toolbox>> to look for new artefact species. New artefact species are those artefacts that are out of scope of the <<Toolbox>> and need to be made in a completely different way with significantly different materials that are not covered by the <<Toolbox>>.
```

Those new artefact species that can be made by slightly modifying test items in the <<Toolbox>> are covered by the normal test plans.

====== New artefact Toolbox updates

> The evaluator is strongly encouraged to report to the BIO-iTC when the evaluator can find the new artefact species that can successfully attack the TOE to add them to the <<Toolbox>>. 

Reviewer: Can we require this instead?

I think we should also have the evaluator report new types of artefacts including those that cannot successfully attack the TOE because if different enough, they may be able to successfully attack other targets.

```suggestion
The evaluator shall report to the BIO-iTC when new artefact species are found so the artefacts may be added to the <<Toolbox>>. 
```
